lib/kin_history.rb

```diff
@@ -0,0 +1,61 @@
+module KinHistory
```

Comment:
Would it be better to make this module a concern named something like `Loggable`? (I truly don't know the answer; it was just my first thought upon seeing `self.included`.)

Reply:
<insert usual quote about naming and caching here>

Would likely be more consistent with existing modules indeed. Blanking at the moment on an adjective that isn't a mouthful like KinChangeTrackable` however.

Reply:
I think just `Loggable` or `UserLoggable` is fine, since it's handling things logged in the `log_items` table. I don't think there's a reason it has to be reserved for FNOK items.

Reply:
There's a bit more we should do when making this module into a concern:

- It should go in `app/models/concerns`, since the model is where we're including our concern. (There's also `app/controllers/concerns` for when you're including the concern module in a controller.)
- It should have `extend ActiveSupport::Concern` on the line after `module UserLoggable`.
- Now we can use an [`included` block](https://api.rubyonrails.org/classes/ActiveSupport/Concern.html#method-i-included) for the callback.